### PR TITLE
Add functionality to set log level for all request logs

### PR DESCRIPTION
**Pull request by nicklas-dohrn**
Adds functionality to set request logging level globally

### CODE REVIEW

File: core/log-core.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ const LOGGING_LEVELS = {
 };
 
 var logLevelInt = 2;
+var reqestLogLevel = "info";
```
Comment (christiand93) [inline core/log-core.js:27]: reqestLogLevel --> requestLogLevel

**Review by christiand93 — CHANGES_REQUESTED** [review]
We should have a small section in README.md to document the new `setRequestLogLevel` function.

**Comment by nicklas-dohrn** [comment]
Added documentation for the new feature

File: README.md [hunk 2]
```diff
@@ -130,6 +130,14 @@ http.createServer((req, res) => {
...
 ```
 
+#### Request log level
+
+You can set the level, with which request logs will be written:
+
+```js
+log.setRequestLogLevel("verbose");
+```
+the default log level for request logs is ``info``
```
Comment (christiand93) [inline README.md:140]:
```suggestion
The default log level for request logs is ``info``.
```

**Review by christiand93 — APPROVED** [review]
LGTM  👍

**Pull request merged**